templates/undercloud.yaml

```diff
@@ -40,7 +42,7 @@ resources:
         - "default"
         - {get_resource: undercloud_sg}
       name: {get_param: undercloud_name}
-      user_data_format: SOFTWARE_CONFIG
```

Comment:
why change the default user_data_format to RAW from SOFTWARE_CONFIG? I think it needs to remain SOFTWARE_CONFIG in order to deploy deployment metadata from Heat.